src/containers/explore/EntityNeighborsContainer.js

```diff
@@ -0,0 +1,177 @@
+  const associationEntitySetsMap = useSelector(selectOrgDataSets(organizationId, associationEntitySetIds));
+
+  const handleAssociationOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: dataSetId } = currentTarget;
+      if (dataSetId) {
+        if (visibleOptions.get(dataSetId, Set()).isEmpty()) {
+          const neighborsDataSetIds :Set = neighbors.get(dataSetId, Map()).keySeq().toSet();
+          setVisibleOptions(visibleOptions.set(dataSetId, neighborsDataSetIds));
+        }
+        else {
+          setVisibleOptions(visibleOptions.delete(dataSetId));
+        }
+      }
+    }
+  };
+
+  const handleNeighborOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: entitySetId } = currentTarget;
+      if (visibleNeighbors.includes(entitySetId)) {
+        setVisibleNeighbors(visibleNeighbors.delete(entitySetId));
+      }
+      else {
+        setVisibleNeighbors(visibleNeighbors.add(entitySetId));
+      }
+    }
+  };
+
+  const associationEntitySetChips = associationEntitySetsMap.valueSeq().map((associationEntitySet :Map) => {
+    const associationEntitySetName = associationEntitySet.get('title') || associationEntitySet.get('name');
+    const associationEntitySetId = associationEntitySet.get('id');
+    return (
+      <Checkbox
+          checked={!visibleOptions.get(associationEntitySetId, Set()).isEmpty()}
+          label={associationEntitySetName}
+          mode="chip"
+          onChange={handleAssociationOnClick}
+          value={associationEntitySetId} />
+    );
+  });
+
+  const neighborDataSetChips = visibleOptions.valueSeq().flatten().toSet().map((neighborESID :UUID) => {
+    const neighborDataSet = dataSetsMap.get(neighborESID, Map());
+    const neighborDataSetName = neighborDataSet.get('title') || neighborDataSet.get('name');
+    const neighborDataSetId = neighborDataSet.get('id');
+    return (
+      <Checkbox
+          checked={visibleNeighbors.includes(neighborDataSetId)}
+          label={neighborDataSetName}
+          mode="chip"
+          onChange={handleNeighborOnClick}
+          value={neighborDataSetId} />
+    );
+  });
+
+  const visibleNeighborTables = visibleOptions.entrySeq().map(([associationESID, neighborESIDs]) => {
+    const associationEntitySet = associationEntitySetsMap.get(associationESID, Map());
+    const associationEntitySetName = associationEntitySet.get('title') || associationEntitySet.get('name');
+    const associationEntitySetId = associationEntitySet.get('id');
+    return visibleNeighbors.some((neighborESID) => neighborESIDs.includes(neighborESID))
+      && (
+        <>
+          <br />
+          <Typography variant="h4">
+            { associationEntitySetName }
```

Comment:
I think this should also use `DataSetTitle` because it is an entity set. Or maybe neither should use it.

<img width="282" alt="Screen Shot 2021-06-23 at 4 38 41 PM" src="https://user-images.githubusercontent.com/440299/123181154-930d0980-d441-11eb-8259-16719e1c0a7f.png">

Reply:
Talked to shireen, and we're gonna do this for now. She's gonna think if there is a better solution.
<img width="1142" alt="Screen Shot 2021-06-25 at 12 53 35 PM" src="https://user-images.githubusercontent.com/19995069/123480664-87435380-d5b7-11eb-852d-e9cf41728eff.png">